src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -1903,16 +1903,10 @@ class GrpcLbFactory : public LoadBalancingPolicyFactory {
 // Plugin registration
 //
 
-void grpc_lb_policy_grpclb_init() {
-  grpc_core::LoadBalancingPolicyRegistry::Builder::
-      RegisterLoadBalancingPolicyFactory(
-          absl::make_unique<grpc_core::GrpcLbFactory>());
-}
-
-void grpc_lb_policy_grpclb_shutdown() {}
-
 namespace grpc_core {
 void RegisterGrpcLbLoadReportingFilter(CoreConfiguration::Builder* builder) {
```

Comment:
Suggest renaming this to `RegisterGrpcLbPolicy()`, since the LB policy is the main thing here, not the filter (the filter is needed only to provide some functionality for the LB policy).

Reply:
fixed